src/utils/components/TabModal/TabModal.tsx

```diff
@@ -69,7 +69,10 @@ const TabModal: TabModalFC = React.memo(
 
       onSubmit(obj)
         .then(onClose)
-        .catch(setError)
+        .catch((submitError) => {
+          setError(submitError);
+          console.error(submitError);
```

Comment:
??

Reply:
I meant why use console.error? we shouldn't use it like we shouldn't leave console.log :)